frontends/web/src/routes/account/send/utxos.tsx

```diff
@@ -56,6 +58,7 @@ export const UTXOs = ({
   const { t } = useTranslation();
   const [utxos, setUtxos] = useState<TUTXO[]>([]);
   const [selectedUTXOs, setSelectedUTXOs] = useState<TSelectedUTXOs>({});
+  const [showAddressReusedMessage, setShowAddressReusedMessage] = useState(0);
```

Comment:
nit: I find the name of this const slightly misleading. Maybe smth like `selectedReusedAddresses`? Since it is a number and not a bool.

Reply:
Updated to `reusedAddressUTXOs`.